app/javascript/listings/listingForm.jsx

```diff
@@ -119,3 +127,17 @@ export default class ListingForm extends Component {
     );
   }
 }
+
+ListingForm.propTypes = {
+  listing: PropTypes.string,
+  categoriesForDetails: PropTypes.string,
+  categoriesForSelect: PropTypes.string,
+  organizationsString: PropTypes.string,
+};
+
+ListingForm.defaultProps = {
+  listing: '',
+  categoriesForDetails: '',
+  categoriesForSelect: '',
+  organizationsString: '',
+};
```

Comment:
shouldn't the props be required? 

I recently reviewed an open PR that's also working on this file and the author there set the props as required https://github.com/thepracticaldev/dev.to/pull/4253/files